### PR TITLE
[Mobile]Move items not in guideline to Etc

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problem] Main menu items not corresponding to the guide
[Solution] Move it to Etc

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
Duplicate PR: https://github.com/Samsung/TAU/pull/949

**Comment by singa2000** [comment]
Oh !
I knew that https://github.com/Samsung/TAU/pull/949 is  only  for wearable profile.
If it's duplicate with, I will be close it.

**Comment by lslachciak** [comment]
I think wearable profile is already done with Etc.

File: examples/mobile/UIComponents/index.html [hunk 1]
```diff
@@ -56,11 +56,6 @@ <h1>
 						Text fields
 					</a>
 				</li>
-				<li class="ui-li-anchor">
-					<a href="components/toast/index.html">
-						Toast
-					</a>
-				</li>
 				<li class="ui-li-anchor">
```
Comment (lslachciak) [inline examples/mobile/UIComponents/index.html:59]: Navigation Elements are not part of current guide. Some of items in it should became part of View controls in future.

**Comment by singa2000** [comment]
This is duplicat with 949. So I closed the PR.

**Pull request closed without merging**